tests/test_openmc_get_optical_thickness.py

```diff
@@ -0,0 +1,73 @@
+import openmc
+from openmc import RegularMesh
+import openmc.lib
+import numpy as np
+
+# Material
+material = openmc.Material(name="Hydrogen")
```

Comment:
We should implement this with fictitious cross-sections that are simple and well-known